src/index.html

```diff
@@ -4,9 +4,299 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img src="images/logo.png"
+             alt="moyo-logo"
+             class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple"
+               class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung"
+               class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones"
+               class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover">Laptops & Computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets"
+               class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets"
+               class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo"
+               class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video"
+               class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div class="catalog">
+        <div class="card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="Apple a1419 imac 27 Retina 5k Monoblock"
+          >
+          <p class="card__desciption">APPLE A1419 iMac 27" Retina 5K Monoblock
+              (MNED2UA/A)
+          </p>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__stars">
+            <div class="stars">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <p class="card__review">Reviews: 5</p>
+          </div>
+          <div class="card__price">
```

Comment:
Check `checklist`
https://github.com/mate-academy/layout_stars/blob/master/checklist.md
```suggestion
      </nav>
    </header>
    
    <main class="container">
      <div class="catalog">
        <div class="card" data-qa="card">
          <img
            class="card__img"
            src="./images/imac.jpeg"
            alt="Apple a1419 imac 27 Retina 5k Monoblock"
          >
          
          <p class="card__desciption">APPLE A1419 iMac 27" Retina 5K Monoblock
              (MNED2UA/A)
          </p>
          
          <p class="card__code">Product code: 195434</p>
          
          <div class="card__stars">
            <div class="stars">
              <span class="stars__star"></span>
              <span class="stars__star"></span>
              <span class="stars__star"></span>
              <span class="stars__star"></span>
              <span class="stars__star"></span>
            </div>
            
            <p class="card__review">Reviews: 5</p>
          </div>
          
          <div class="card__price">
```